src/Native/Utils.js

```diff
@@ -325,12 +325,17 @@ Elm.Native.Utils.make = function(localRuntime) {
 		throw new Error('Runtime error in module ' + moduleName + ' (' + span + ')' + msg);
 	}
 
+	function formatValue(value) {
```

Comment:
Agree about this, one way or the other. After the change, @rtfeldman, please ping this thread for merge :)